src/components/WebcamCapture/WebcamCapture.js

```diff
@@ -16,15 +13,15 @@ class WebcamCapture extends Component {
 	webcam = React.createRef();
 
 	requestScreenshot = () => {
-		return new Promise((resolve, reject) => {
+		return new Promise(() => {
```

Comment:
I don't think this currently works since we're not resolving the image.

We can remove the Promise wrapper since we're returning a promise directly